client/src/metrics.rs

```diff
@@ -113,6 +122,7 @@ impl MetricsContext {
 const KEY_NAMESPACE: &str = "namespace";
 const KEY_SVC_METHOD: &str = "operation";
 const KEY_TASK_QUEUE: &str = "task_queue";
+const KEY_STATUS_CODE: &str = "status_code";
```

Comment:
Can you update the original features issue with details on the exact key name we're using, that we're using the proper screaming case enum from https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto, and which metrics we're adding it on? This can help other SDKs make sure they conform.

Reply:
Sure, this is all exactly what Java is doing now